deploy/kubernetes/elasticsearch/elasticsearch.gcloud.yaml

```diff
@@ -237,4 +237,4 @@ spec:
       targetPort: 9200
   selector:
     elasticsearch.k8s.elastic.co/cluster-name: elasticsearch
-    elasticsearch.k8s.elastic.co/node-data: "true"
\ No newline at end of file
+    elasticsearch.k8s.elastic.co/node-data: "true"
```

Comment:
I think my text editor added the newline here… Happy to remove this if we don't want a final newline